packages/agoric-cli/src/commands/psm.js

```diff
@@ -153,16 +153,13 @@ export const makePsmCommand = logger => {
     .option('--giveMinted [DOLLARS]', 'amount of minted tokens to give', Number)
     .option('--feePct [%]', 'Gas fee percentage', Number)
     .option('--offerId [string]', 'Offer id', String, `swap-${Date.now()}`)
-    .action(async function (opts) {
+    .action(async function (
```

Comment:
NOT URGENT comment:

Why do we still have `function` functions?

Reply:
resolving as outside scope